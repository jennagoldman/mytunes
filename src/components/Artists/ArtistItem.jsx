import React from 'react';
import PropTypes from 'prop-types';

const ArtistItem = ({ artistName }) => (
  <>
    <p>{artistName}</p>
  </>
);

ArtistItem.propTypes = {
  artistName: PropTypes.string.isRequired
};


export default ArtistItem;
