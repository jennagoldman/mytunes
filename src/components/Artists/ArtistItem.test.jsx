import React from 'react';
import { shallow } from 'enzyme';
import ArtistItem from './ArtistItem.jsx';

describe('ArtistsList', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<ArtistItem artistName={'Electric Light Orchestra'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
