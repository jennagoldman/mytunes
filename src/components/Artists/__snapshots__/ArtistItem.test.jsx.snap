// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtistsList matches a snapshot 1`] = `
<p>
  Electric Light Orchestra
</p>
`;
